javascript/package.json

```diff
@@ -11,6 +11,7 @@
     "seedrandom": "^2.4.3"
   },
   "devDependencies": {
-    "jasmine": "^3.1.0"
+    "jasmine": "^3.1.0",
+    "prettier": "^1.12.0"
```

Comment:
Code readability was adding friction to refactoring, so brought in prettier to clean it up. No script yet, just `$(yarn bin)/prettier --write src/**/*.js`.